robots/issue-creator/BUILD.bazel

```diff
@@ -7,7 +7,11 @@ go_binary(
     visibility = ["//visibility:public"],
 )
 
-prow_image(name = "image")
+prow_image(
+    name = "image",
+    base = "@alpine-base//image",
```

Comment:
Can we no longer use a distroless image?

Reply:
We should be able to. This was mainly a copy from another image.

Updated, PTAL.